Sources/SourceKitLSP/Swift/DocumentSymbols.swift

```diff
@@ -170,6 +170,16 @@ fileprivate final class DocumentSymbolsFinder: SyntaxAnyVisitor {
     return .skipChildren
   }
 
+  override func visit(_ node: DeinitializerDeclSyntax) -> SyntaxVisitorContinueKind {
+    return record(
+      node: node,
+      name: node.deinitKeyword.text,
+      symbolKind: .null,
```

Comment:
Kind of weird, but could make constructor?

Reply:
I thought about it as well but then felt that `constructor` is weird because it’s not a constructor… But I don’t have strong opinions here.

Reply:
I just checked and VS Code at least uses the same icon for constructor and function, so it’s odd if `deinit` uses a different icon. `clangd` uses `constructor`, so let’s just do that as well.


<img width="601" alt="Screenshot 2024-06-03 at 09 07 41" src="https://github.com/apple/sourcekit-lsp/assets/4062178/8e4426b8-d158-4bd9-8f76-42880e754516">